src/selectors/findDescendant.ts

```diff
@@ -12,10 +12,14 @@ const findDescendant = (state: State, thoughtId: ThoughtId | null, values: strin
   if (!thoughtId || values.length === 0) return thoughtId
   if (!Array.isArray(values)) values = [values]
   // if the value is a meta attribute, use childrenMap for O(1) lookup
+
+  const currentThought = getThoughtById(state, thoughtId)
```

Comment:
Since the refactoring in this file is not relevant to the purpose of the PR, I would prefer see it in a separate PR.

However, it is a minor change (and a good one!), so I would be okay with this.